Description: Remove checks for individual cardsets' COPYRIGHT files
 which we remove in order to comply with Debian policy.
Author: Bernhard Reiter <ockham@raz.or.at>
Forwarded: not-needed

--- a/pysollib/app.py	2018-07-31 11:44:58.127291946 +0800
+++ b/pysollib/app.py	2018-07-31 11:46:17.375162990 +0800
@@ -1439,11 +1439,10 @@
                     d = os.path.join(dir, name)
                     if not os.path.isdir(d):
                         continue
-                    f1 = os.path.join(d, "config.txt")
-                    f2 = os.path.join(d, "COPYRIGHT")
-                    if os.path.isfile(f1) and os.path.isfile(f2):
+                    f = os.path.join(d, "config.txt")
+                    if os.path.isfile(f):
                         try:
-                            cs = self._readCardsetConfig(d, f1)
+                            cs = self._readCardsetConfig(d, f)
                             if cs:
                                 # from pprint import pprint
                                 # print cs.name
@@ -1458,7 +1457,7 @@
                                     # print '+', cs.name
                             else:
                                 print_err('fail _readCardsetConfig: %s %s'
-                                          % (d, f1))
+                                          % (d, f))
                                 pass
                         except Exception:
                             # traceback.print_exc()
--- a/pysollib/tile/selectcardset.py	2018-07-31 11:44:58.127291946 +0800
+++ b/pysollib/tile/selectcardset.py	2018-07-31 12:08:12.030780247 +0800
@@ -525,15 +525,12 @@
         frame.columnconfigure(0, weight=1)
         frame.rowconfigure(1, weight=1)
         #
-        text = ''
-        f = os.path.join(cardset.dir, "COPYRIGHT")
-        try:
-            text = open(f).read()
-        except Exception:
-            pass
-        if text:
-            text_w.config(state="normal")
-            text_w.insert("insert", text)
+        text = '''\
+Copyright and license information for all cardsets and
+other game components can be found in the file
+/usr/share/doc/pysolfc/copyright
+'''
+        text_w.insert("insert", text)
         text_w.config(state="disabled")
         #
         focus = self.createButtons(bottom_frame, kw)
--- a/pysollib/tk/selectcardset.py	2018-07-31 11:44:58.127291946 +0800
+++ b/pysollib/tk/selectcardset.py	2018-07-31 12:08:56.859500724 +0800
@@ -488,15 +488,12 @@
         frame.columnconfigure(0, weight=1)
         frame.rowconfigure(1, weight=1)
         #
-        text = ''
-        f = os.path.join(cardset.dir, "COPYRIGHT")
-        try:
-            text = open(f).read()
-        except Exception:
-            pass
-        if text:
-            text_w.config(state="normal")
-            text_w.insert("insert", text)
+        text = '''
+Copyright and license information for all cardsets and
+other game components can be found in the file
+/usr/share/doc/pysolfc/copyright
+'''
+        text_w.insert("insert", text)
         text_w.config(state="disabled")
         #
         focus = self.createButtons(bottom_frame, kw)
